web/wp-content/themes/dctx-v2/docs/assets/js/vendor/prism.min.js

```diff
@@ -0,0 +1,9 @@
+/* http://prismjs.com/download.html?themes=prism&languages=markup+css+css-extras+clike+javascript+scss */
```

Comment:
:warning: **Warning**: File appears to be minified and cannot be processed (*Internal.Tokenizer.Exception*).